src/js/angular/guides/steps/core/plugin.js

```diff
@@ -174,10 +180,10 @@ PluginRegistry.add('guide.step', [
                 content: options.content || 'guide.step_plugin.guide-ended.content',
                 lastStep: true,
                 show: (guide) => () => {
-                    guide.options.confirmCancel = false;
+                    // guide.options.confirmCancel = false;
```

Comment:
Document why this is commented out or delete it

Reply:
Forgot to remove